components/component_style/button.css

```diff
@@ -0,0 +1,66 @@
+.btn {
+    padding: var(--btn-padding);
+    cursor: pointer;
+    text-transform: uppercase;
+    font-weight: var(--bold-font-weight);
+    border-radius: var(--default-border);
+    margin: var(--default-spacing);
+}
+
+.btn-primary-contained {
+    border: 2px solid var(--primary-color);
+    background-color: var(--primary-color);
+    color: white;
+    box-shadow: var(--default-shadow);
+}
+
+.btn-primary-contained:hover {
+    background-color: var(--primary-color-dark-shade1);
+}
+
+.btn-primary-outlined {
+    border: 2px solid var(--primary-color);
+    background-color: white;
+    color: var(--primary-color);
+}
+
+.btn-primary-outlined:hover {
+    background-color: #f7f7f7;
```

Comment:
This can also be a color variable as well.

Reply:
thank you for pointing out, updated this as variable but forgot to update here